dev-util/dwarves/dwarves-9999.ebuild

```diff
@@ -1,12 +1,12 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 EGIT_REPO_URI="https://git.kernel.org/pub/scm/devel/pahole/pahole.git"
 
-PYTHON_COMPAT=( python3_6 )
-inherit multilib cmake-utils git-r3 python-single-r1
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit cmake git-r3 multilib python-single-r1
```

Comment:
You don't need multiline for get_libdir anymore.

Reply:
Thanks for spotting this. I would like to ask whether there is a tool (repoman?) to warn about redundant imports in ebuilds - for example the programming language Go will refuse to compile files containing unused import directives (unless an import directive uses a special syntax).

Reply:
I don't think so, although repoman and pkgcheck may spot some (not sure).